framework/web/ErrorHandler.php

```diff
@@ -391,8 +391,8 @@ public function argumentsToString($args)
                 $args[$key] = '<span class="keyword">' . ($value ? 'true' : 'false') . '</span>';
             } elseif (is_string($value)) {
                 $fullValue = $this->htmlEncode($value);
-                if (mb_strlen($value, 'utf8') > 32) {
-                    $displayValue = $this->htmlEncode(mb_substr($value, 0, 32, 'utf8')) . '...';
+                if (mb_strlen($value, Yii::$app->charset) > 32) {
+                    $displayValue = $this->htmlEncode(mb_substr($value, 0, 32, Yii::$app->charset)) . '...';
```

Comment:
The error handler should not rely on an application to be existing. I think in this case it is fine to assume utf8